src/VirtoCommerce.CustomerExportImportModule.Web/module.manifest

```diff
@@ -5,6 +5,7 @@
     <version-tag></version-tag>
     <platformVersion>3.53.0</platformVersion>
     <dependencies>
+        <dependency id="VirtoCommerce.Customer" version="3.28.0" />
```

Comment:
Customer and Store modules were added as deps.

Reply:
Fixed